README.md

```diff
@@ -13,6 +13,12 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-148-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+<br/>
+
+Read in different language : [![FR](/assets/flags/FR.png)**FR**](/fr/README.md),
+
+<br/>
+
```

Comment:
If we plan to include this for French then we must include all other languages already available like Korean and Chinese

Reply:
@iluwatar What are your suggestions on this?

Reply:
hello @ohbus yes we must include chinese and korean in this part :)

Reply:
done i have added chinese and korean flages in main README

Reply:
Thank you.